### PR TITLE
Un 387 frontend record revealed

### PR DESCRIPTION
**Pull request by William-Blackie**
Related ticket(s):
- <- https://national-archives.atlassian.net/browse/UN-204 ->

## About these changes
Here I've added a image gallery with tabbs for JS users and a CSS only version of non-js users. I've used Axe dev'tools to check and I'm not getting any warnings specifically for this feature but would appreciate @janehughes input when you have a moment.

## How to check these changes
You'll find these changes in on a CloserLookPage type, I made one under the CollectionExplorer, input the record revealed fields and this will populate the FE. 

Tab from the start of the doccument and you will be able to; open/close the gallery, select and change tabs, focus the content.

For non-js users (Disable this in your browser) you will see that the I've rendered the tabs one after another as that seemed best to fit the content, although no design is provided for this. Happy for a screenshare.

Thanks - Will.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [ ] Ensured that PR includes only commits relevant to the ticket.
- [ ] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## For Reviewer

- Before merging PR

    `main` branch :

        - check PR merge tile begins with `release/<major.minor.patch>:<default/custom description>`
        - check PR merge tile begins with ticket number Ex `DF-XXX: Ticket name / short description`
        - select `Create a merge commit`

    `develop` branch:

        - check if merges into base branch are `not to be kept On Hold` and then proceed.
        - check PR merge tile begins with ticket number Ex `DF-XXX: Ticket name / short description`
        - select `Squash and merge`

## For Reviewer/Developer

- After merging PR:

  - check and arrange to deploy on platform-sh.
  - update status on JIRA ticket on deployment.
  - update slack on deployment progress of the JIRA ticket once deployed.

### CODE REVIEW

File: templates/static/cookie-consent/ds-cookie-consent.css.map [hunk 1]
```diff
@@ -1 +0,0 @@
-{"version":3,"sourceRoot":"","sources":["../src/ds-cookie-consent.scss"],"names":[],"mappings":"AACE;EACE;;AACA;EACE;;AACA;EAFF;IAGI;;;AAKN;EACE;EACA;;AAGF;EACE;;AAGF;EACE;;;AAIJ;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AACA;EACE;EACA;EACA;EACA;;AACA;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;;AACA;EACE;EACA;EACA;EACA;EACA;;AAEF;EACE;EACA;;AACA;EACE;EACA;EACA;EACA;EACA;;AAEF;EACE;;AAIN;AAAA;AAAA;EAGE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AACA;AAAA;AAAA;AAAA;AAAA;EAEE;EACA;EACA;;AAEF;AAAA;AAAA;EACE;;AAEF;EAvBF;AAAA;AAAA;IAwBI;IACA;IACA;;;AAGJ;EACE;EACA;EACA;EACA;;AACA;EAEE;EACA;EACA;EACA;;AAEF;EACE;;AAEF;EAfF;IAgBI;;;AAEF;EAlBF;IAmBI;;;AAGJ;EA/FF;IAgGI;;;;AAMJ;EAEI;IACE;;;;AAON;EACE;;AAEF;EACE","file":"ds-cookie-consent.css"}
```
Comment (William-Blackie) [inline templates/static/cookie-consent/ds-cookie-consent.css.map:1]: Technically out of scope, but I've added a line to the .gitignore so we're not commiting css maps.

**Comment by JohnHeeryTNA** [comment]
Hi @William-Blackie and @sdwilkes This is looking really good but a few things to address:

I think we need a field in the backend to populate the **hero image**
Screenshot 2023-02-02 at 15 08 07
Apologies if I'm misunderstanding the cms side of things.

-------

**Translation/Transcription tabs** work well, however we may need to address the font size of titles on Tablet/Small desktop view as they stack at 1400 > 986px 

Screenshot 2023-02-02 at 15 12 12

-------
Can we have the cursor become the 'hand' icon on all buttons on hover state please? Currently an arrow.

Screenshot 2023-02-02 at 15 29 59

-------

Do we want to include a +/- svg icon for the 'read more'/'read less' cta as per design? @sdwilkes ?

Screenshot 2023-02-02 at 15 13

-------

These buttons currently link in the same window/tab, do we want to have these open a new tab? i.e : target="_blank"? 
cta-buttons

-------
 For non js users can we hide the 'Read less' link please?
 
 non-js

I'm still doing a full code review in terms of html validation/semantics but confidence is high. Be good to get @janehughes feedback too on any usability concerns

**Comment by sdwilkes** [comment]
@JohnHeeryTNA @William-Blackie 

Yes, let's use an SCG for plus/minus. The ones in the Fontawesome library might work well.

Let's keep the links in the same window as they're linking to TNA branded destinations.

Thanks.

**Comment by William-Blackie** [comment]
> Hi @William-Blackie and @sdwilkes This is looking really good but a few things to address:
> 
> I think we need a field in the backend to populate the **hero image** Screenshot 2023-02-02 at 15 08 07 Apologies if I'm misunderstanding the cms side of things.
> 
> **Translation/Transcription tabs** work well, however we may need to address the font size of titles on Tablet/Small desktop view as they stack at 1400 > 986px
> 
> Screenshot 2023-02-02 at 15 12 12
> 
> Can we have the cursor become the 'hand' icon on all buttons on hover state please? Currently an arrow.
> 
> Screenshot 2023-02-02 at 15 29 59
> 
> Do we want to include a +/- svg icon for the 'read more'/'read less' cta as per design? @sdwilkes ?
> 
> Screenshot 2023-02-02 at 15 13
> 
> These buttons currently link in the same window/tab, do we want to have these open a new tab? i.e : target="_blank"? cta-buttons
> 
> For non js users can we hide the 'Read less' link please?
> 
> non-js
> 
> I'm still doing a full code review in terms of html validation/semantics but confidence is high. Be good to get @janehughes feedback too on any usability concerns

Thanks for reviewing, I'll pick up these changes. I've made a small mistake by mixing the name conventions / tickets, this work only covered the record-revealed image gallery, happy to role the other comments into this though but I appolgise for passing what looked like I was asking to review un-finished work.

**Comment by JohnHeeryTNA** [comment]
No problem @William-Blackie - these are just the bits I think need addressing for the whole page, thanks.

**Comment by LeanneEmans** [comment]
Hello @janehughes  please may I kindly ask if you've had a chance to review this 'Record Revealed' Front End Dev PR from an Accessibility view? Thank you!

**Review by janehughes — COMMENTED** [review]
Thanks Will I've added some initial comments. As I mentioned yesterday I'm going to have to get my local build issues sorted before I can do a more thorough a11y/UX review so I've held off adding any comments on that just yet.

**Pull request closed without merging**